src/extensions/replay/sessionrecording.ts

```diff
@@ -346,6 +357,11 @@ export class SessionRecording {
     }
 
     afterDecideResponse(response: DecideResponse) {
+        const receivedSampleRate = response.sessionRecording?.sampleRate
+        const parsedSampleRate = _isNullish(receivedSampleRate) ? null : parseFloat(receivedSampleRate)
+
+        const receivedMinimumDuration = response.sessionRecording?.minimumDurationMilliseconds
```

Comment:
Curious why `parsedSampleRate` does `_isNullish` and `parseFloat` but `receivedMinimumDuration` only `_isUndefined`.
Aren't they both the very same data type?

Reply:
Ah, I didn't remember either. (I think because sample rate is a float) it's a string when received and to pass it to `parseFloat` we need to know it's neither null nor undefined. Slightly yucky but 🤷